source/slang/slang-ir-specialize.cpp

```diff
@@ -597,7 +606,7 @@ struct SpecializationContext
     {
         // Note: While we currently have named the instruction
         // `lookup_witness_method`, the `method` part is a misnomer
-        // and the same instruction can look up *any* interface
+        // and the same instruction can look up *any* interfacemay
```

Comment:
Typo.
> interface may
With a space character in the middle.

Reply:
Fixed.